plugins/modules/xbps.py

```diff
@@ -67,6 +67,26 @@
         type: bool
         default: true
         version_added: '0.2.0'
+    root:
+        description:
+            - The full path for the target root directory.
+        type: path
+        version_added: '10.1.0'
```

Comment:
10.1.0 has been released, please update (here and in the other placea in the files):
```suggestion
        version_added: '10.2.0'
```